pkgs/by-name/sn/snakemake/package.nix

```diff
@@ -23,8 +27,11 @@ python3.pkgs.buildPythonApplication rec {
       --replace-fail '"unit_tests/templates"' '"'"$PWD"'/snakemake/unit_tests/templates"'
   '';
 
-  propagatedBuildInputs = with python3.pkgs; [
+  build-system = [];
```

Comment:
https://github.com/snakemake/snakemake/blob/1874fa86b1ebf52ece6646f32d4a3ed025499183/pyproject.toml#L1-L6

Reply:
I added `setuptools`.
As `snakemake` will always use the `nixpkgs` "default" python version (i.e. 3.12 currently), I don't think that we need to handle the `< python 3.11` use case.

Reply:
If this works out for you, feel free to merge :)